public/assets/js/scripts.js

```diff
@@ -1,38 +1,63 @@
-var fieldValidation, validation;
-
-fieldValidation = function(checkThis) {
-  var $checkThis, isValid;
-  isValid = true;
-  $checkThis = $(checkThis);
-  if ($.trim($checkThis.val()).length) {
-    $checkThis.removeClass('invalid');
-  } else {
-    isValid = false;
-    $checkThis.addClass('invalid');
-  }
-  return isValid;
-};
-
-validation = function() {
-  $('form').submit(function() {
-    var $inputs, isValid;
-    isValid = true;
-    $inputs = $(this).find('[required]');
-    $inputs.each(function() {
-      var $input;
-      $input = $(this);
-      if (!fieldValidation($input)) {
-        isValid = false;
+const validation = function() {
+  $('form').validate({
+    rules: {
+      name: "required",
+      url: {
+        required: true,
+        url: true
+      },
+      category: "required",
+      duration: {
+        required: true,
+        min: 1,
+        number: true
+      },
+      summary: "required",
+      location: "required",
+      learn_more: {
+        required: true,
+        url: true
+      },
+     apprentice: {
+        required: true,
+        email: true
+      },
+      email: {
+        required: true,
+        email: true
+      }
+    },
+    messages: {
+      compensation: "Your apprenticeship must be paid",
+      name: "Please enter your company name",
+      url: {
+        required: "Please enter your company website",
+        url: "Please enter a valid URL"
```

Comment:
For the URL I think we should explicitly mention that all URLs need the `http:/` prefix, its slightly confusing when a user enters a valid URL but without the prefix and it still throws the error.

Reply:
good call!